assets/src/data/model/event/index.js

```diff
@@ -87,29 +87,36 @@ export const mapOrderBy = ( orderBy ) => {
  *
  * @param {boolean} showExpired  Whether or not to include expired events.
  * @param {string} categorySlug  Return events for the given categorySlug
- * @param {string} month         Return events for the given month.  Can be any
- *                                 in any month format recognized by moment.
- * @return {string}             The assembled where conditions.
+ * @param {string} month         Return events for the given month.
+ * 								 Can be any month format recognized by moment.
+ * @return {string}              The assembled where conditions.
  */
-export const whereConditions = ( { showExpired = true, categorySlug, month = 'none' } ) => {
+export const whereConditions = ( {
+	showExpired = false,
+	categorySlug,
+	month = 'none',
+} ) => {
 	const where = [];
 	const GREATER_AND_EQUAL = encodeURIComponent( '>=' );
 	const LESS_AND_EQUAL = encodeURIComponent( '<=' );
 
 	if ( ! showExpired ) {
-		where.push( 'where[Datetime.DTT_EVT_end**expired][]=>&where[Datetime.DTT_EVT_end**expired][]=' +
-			nowDateAndTime.local().format() );
+		where.push(
+			'where[Datetime.DTT_EVT_end**expired][]=>' +
```

Comment:
huh, I think I've missed this in multiple places (i.e. will affect our datetime and ticket implementation as well).  This line results in (with the other two):

`where[Datetime.DTT_EVT_end**expired][]=>&where[Datetime.DTT_EVT_end**expired[]='{some-date-string}'`

It doesn't makes sense?  I think the first part which is `where[Datetime.DTT_EVT_end**expired][]=>&` shouldn't even be present right?

Reply:
actually no, it's correct.  But we should be using `GREATER_AND_EQUAL` instead of `=>` in that line.

Reply:
ya I never grokked that, but assumed it was something funny with the way the REST API worked

so... the correct line should be:

```
'where[Datetime.DTT_EVT_end**expired][]=' + GREATER_AND_EQUAL  +
```

or

```
'where[Datetime.DTT_EVT_end**expired][]' + GREATER_AND_EQUAL  +
```

I would think the former would be correct ( ie: replace `=>` with `=' + GREATER_AND_EQUAL` ),
but it sounds like you are saying the latter  ( ie: replace `=>` with GREATER_AND_EQUAL` ),
which makes no sense at all to me.

Reply:
OH, actually maybe its supposed to be `=` + `>` only.  In which case it'd probably be good to add `GREATER_THAN` and `LESS_THAN` constants that encode the equal and less than characters.  Since multiple models will likely be using these constants, it'd probably also be good to move them into the `data/model/base.js` module and they can just be included where needed?

Reply:
kk no THAT makes sense and I totally agree.
will do that right now